msteams-platform/publishing/apps-localization.md

```diff
@@ -2,32 +2,81 @@
+## Localizing your AppSource listing
 
-### Add English language
+If you're publishing to the app store, you need to be aware that localizing your AppSource listing is not yet supported. However, in preparation for support for localized listings in the app store you can add additional languages to your listing. Currently only the default (English) language information you provide in [Seller Dashboard](http://go.microsoft.com/fwlink/?LinkId=248605) for your listing will appear in the [AppSource website](https://appsource.microsoft.com/marketplace/apps?product=office%3Bteams&page=1) listing for your app.
 
-* Fill in the app name.
-* Fill in a short description of the app in English.
-* Fill in the long description of the app in English.
-* In the long description, please also add the line “This app is in “French”.
-* Upload the images of your app UI. The screenshots will show the actual language (in this example, French).
+To configure an additional language for your app, in [Seller Dashboard](http://go.microsoft.com/fwlink/?LinkId=248605), select both English and the additional language of the app. French is used in this example.
 
-### Add French language
+1. Add English language
+    * Fill in the app name.
+    * Fill in a short description of the app in English.
+    * Fill in the long description of the app in English.
+    * In the long description, please also add the line “This app is available in “French”.
+    * Upload the images of your app UI (in English).
+2. Add French language
+    * Fill in the app name.
+    * Fill in a short description of the app in French.
+    * Fill in the Long description of the app in French.
+    * Upload the images of your app UI (in French).
 
-* Fill in the app name.
-* Fill in a short description of the app in French.
-* Fill in the Long description of the app in French.
-* Upload the images of your app UI.
+The images you upload with the English language will be the ones used in AppSource.
 
-The images for both English and French languages will be the same.
+## Localizing the strings in your app manifest
 
-See the [Localized content](~/publishing/office-store-checklist#localized-content) section of the AppSource checklist for more information on these fields.
+In your app manifest file you provide multiple strings that are displayed to the end-user. If you choose, you can provide additional .json files with translations of these strings. Teams will use the translation that matches your end-user's default language settings in their client if it is available. Teams will also use the closest base language to any specific dialect. For example, if the client default language is set to `fr-ca` (French Canadien) and you do not provide a `fr-ca` translation but you do provide a `fr` (French) translation, we will use the `fr` translation.
```

Comment:
and you do not provide a `fr-ca**` translation but you do provide a** `fr` (French) translation, we will use the `fr` translation.

I think that part should read something like but if you do... sounds like there is a missing word while reading that